src/components/CohortBuilder/Summary/index.js

```diff
@@ -96,51 +101,57 @@ const Summary = ({ theme, sqon, api }) => (
         <Row nogutter> no data</Row>
       ) : (
         <Row nogutter>
-          <Col sm={12} md={12} lg={9}>
+          <Col sm={12} md={12} lg={12} xl={9}>
```

Comment:
I think you can omit some of those.

If react-grid-system really works like bootstrap, those parameters are to be interpretet this way:
`xs={12}` means `12 columns when xs _or bigger_, unless overrided by a larger screenClass
`md={6}` means `9 columns when sm _or bigger_, unless overrided by a larger screenClass

So the above example would yoeld:
xs=12, sm=12, md=6, lg=6, xl=6

Reply:
I'll try that out, thanks! :)